recipe/bld.bat

```diff
@@ -1,46 +1,37 @@
-@echo on
+copy "%RECIPE_DIR%\bld_win.sh" .
 
-:: Build Static library (.lib)
-cmake -GNinja ^
-      -DCMAKE_INSTALL_PREFIX="%LIBRARY_PREFIX%" ^
-      %COMPILER% ^
-      -DCMAKE_C_USE_RESPONSE_FILE_FOR_OBJECTS:BOOL=FALSE ^
-      -DBUILD_SHARED_LIBS=OFF ^
-      -DCMAKE_BUILD_TYPE=Release ^
-      --debug-trycompile ^
-      .
-ninja
-if errorlevel 1 exit /b 1
+@REM Use the MSVC sln to build static lib and dll
+msbuild ^
+  /p:Platform=%PLATFORM% ^
+  /p:Configuration=Release ^
+  /p:AdditionalIncludeDirectories=%LIBRARY_INC% ^
+  /p:AdditionalDependencies=/LIBPATH:%LIBRARY_LIB% ^
+  /p:WindowsTargetPlatformVersion=10.0.17763.0 ^
+  windows\vs2017\xz_win.sln
```

Comment:
I can see these variables change in the future. Especially "vs2017" could become a variable instead of hard-coding it everywhere.

Reply:
Good call. I used the existing env variable for all of the `vs2017` entries. I didn't find an existing variable for the SDK version, but since that is only in one spot, I left it for now.